scripts/i3lock-next

```diff
@@ -52,8 +52,14 @@ trap 'rm -f ${_image}' EXIT
 _i3lock_next_helper="${PREFIX}/${LIBDIR}/i3lock-next/i3lock-next-helper"
 _datetimecolor=$($_i3lock_next_helper ${_image} "${_font}"/"${_size}" "${_prompt}") # oops
 
-# get some values we need
+# get image width & height
 _wxh="$(${PREFIX}/${LIBDIR}/i3lock-next/image-size ${_image})"
+# get lock image size to calculate appropriate ring radius
+_lock_wxh="$(${PREFIX}/${LIBDIR}/i3lock-next/image-size ../share/i3lock-next/lock-dark.png)"
+_lock_w="${_lock_wxh%x*}"
+_lock_h="${_lock_wxh#*x}"
+_temp=$((_lock_w*_lock_w + _lock_h*_lock_h))
+_ring_radius="$(echo "$_temp" | python -c "import math; print int(1.1*math.sqrt(float(raw_input()))/2)")"
```

Comment:
I'm not the most familiar with `bash`, so it's news to me that it's limited to integer math. I piped it to `python` so that I could (i) take a square root and (ii) cast back to int. I'm completely open to any suggestions on how to do this better.